wagtailvideos/models.py

```diff
@@ -251,7 +249,7 @@ def video_tag(self, attrs):
             sources.append("<source src='{0}' type='video/{1}' >".format(transcode.url, transcode.media_format.name))
         sources.append("<p>Sorry, your browser doesn't support playback for this video</p>")
         return mark_safe(
-            "<video {0} {1}>{2}</video>".format(attrs, flatatt(kw_attrs), "\n".join(sources)))
+            "<video {0}{1}>{2}</video>".format(attrs, flatatt(kw_attrs), "\n".join(sources)))
```

Comment:
Very minor nit-pick, but this will end up looking like:
```html
<video ...><source ...>
<source ...>
<source ...></video>
```
If you changed the format strings to be like `"<video{attrs}>\n{sources}\n</video>"` then the new lines would be correctly distributed.